Classes/Command/FakeStorageCommandController.php

```diff
@@ -192,4 +192,28 @@ protected function createFakeFiles(int $storageId, string $path = '/')
             unset($file);
         }
     }
+
+    /**
+     * List all storages
+     *
+     * @return void
+     */
+    public function ListStoragesCommand(): void
+    {
+        /** @var QueryBuilder $queryBuilder */
+        $queryBuilder = GeneralUtility::makeInstance(ConnectionPool::class)
+            ->getQueryBuilderForTable('sys_file_storage');
+
+        $storages = $queryBuilder->select('uid', 'name', 'driver', 'tx_fakefal_enable')
+            ->from('sys_file_storage')
+            ->execute()->fetchAll();
+
+        $this->output->output('uid   |   name   |   driver   |   status' . PHP_EOL);
+
+        foreach ($storages as $storage) {
+            $isFakeStorage = $storage['tx_fakefal_enable'] === 1 ? 'fake-storage' : 'real storage';
+            $this->output->output($storage['uid'] . '   |   ' . $storage['name'] . '   |   ' . $storage['driver'] . '   |   ' . $isFakeStorage . '   |   ' . PHP_EOL);
```

Comment:
There's a method `outputLine` we should use here (didn't know about it either until now).
And for the output of tabular data I'd suggest using a format string and arguments:
```
$this->output->outputLine('|%5.5s |%-30.30s | %10.10s | %5.5s |', [$storage['uid'], $storage['name'], $storage['driver'], $isFakeStorage ? 'true' : 'false']);
```

Reply:
Update: there's even a `\TYPO3\CMS\Extbase\Mvc\Cli\ConsoleOutput::outputTable` method to use here ;-)